tests/reducer-test.js

```diff
@@ -19,6 +20,65 @@ const defaultState = {
   currentGoal: null,
 };
 
+tape('back button goes back to previous step', (t) => {
+
+  const stateGoalsList = {
+    ...defaultState,
+    step: steps.GOALS_LIST,
+  };
+  const stateAddGoal = {
+    ...defaultState,
+    step: steps.ADD_GOAL,
+    previousStep: steps.GOALS_LIST,
+  };
+  const stateViewGoal = {
+    ...defaultState,
+    step: steps.VIEW_GOAL,
+    previousStep: steps.GOALS_LIST,
+  };
+  const stateRateGoalFromViewGoal = {
+    ...defaultState,
+    step: steps.RATE_GOAL,
+    previousStep: steps.VIEW_GOAL,
+  };
+  const stateRateGoalFromGoalsList = {
+    ...defaultState,
+    step: steps.RATE_GOAL,
+    previousStep: steps.GOALS_LIST,
+  };
+  const stateFeedback = {
+    ...defaultState,
+    step: steps.FEEDBACK,
+    previousStep: steps.RATE_GOAL,
+  };
+
+  t.equal(backStep(stateGoalsList).step, steps.GOALS_LIST, `stays on same step
```

Comment:
okay these descriptions do not work, ie see below: `previous step should stay null` ->  assertion does not describe *what* makes this happen..